src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -369,7 +369,7 @@ class DistributedClosestPointImpl
     , m_rank(-1)
     , m_nranks(-1)
   {
-    setAllocatorID();
+    setDefaultAllocatorID();
 
     setMpiCommunicator(MPI_COMM_WORLD);
```

Comment:
Should this be changed to allow user to pass their own MPI communicator? I foresee cases when codes are using this in 'multiprob' execution mode.

Reply:
@gunney1 already added that in #903 
and it resolved the issue where we couldn't run more than one test of this example at the same time.